.github/workflows/image-syncer.yml

```
@@ -70,22 +90,16 @@ jobs:
           workload_identity_provider: ${{ vars.GH_COM_KYMA_PROJECT_GCP_WORKLOAD_IDENTITY_FEDERATION_PROVIDER }}
           token_format: "access_token"
           request_reason: "${{ github.repository_owner }}/${{ github.repository }}/${{ github.workflow }}/${{ github.run_id }}"
-          service_account: "image-syncer-reader@sap-kyma-prow.iam.gserviceaccount.com"
-      
-      - name: Set read-only GCP credentials
-        id: set_gcp_credentials
-        run: |
-          chmod 444 $GOOGLE_APPLICATION_CREDENTIALS
-          ls -ld $GOOGLE_APPLICATION_CREDENTIALS
+          # service_account is required for the token_format "access_token"
+          service_account: ${{ steps.set_image_syncer_gcp_service_account.outputs.IMAGE_SYNCER_SA_EMAIL }}
```

```suggestion
          service_account: ${{ steps.get_image_syncer_gcp_service_account.outputs.IMAGE_SYNCER_SA_EMAIL }}
```

Shouldn't it be like that?